CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Added prop for the event `CommentChange`.
+
 ### Fixed
+- Added `CommentChange` to the list of events that trigger the onEditorChange handler. 
```

Comment:
```suggestion
- Added `CommentChange` to the list of events that trigger the `onEditorChange` handler. 
```
Not sure of the normal style here, but as it's a function name I figured it should be wrapped in backticks 🤷

Reply:
Agreed.